mlrun/wasbfs/fs.py

```diff
@@ -0,0 +1,175 @@
+
+    def glob(self, path, **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.glob(az_path, **kwargs)
+
+    def ls(
+        self,
+        path: str,
+        detail: bool = False,
+        invalidate_cache: bool = False,
+        delimiter: str = "/",
+        return_glob: bool = False,
+        **kwargs,
+    ):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.ls(
+            az_path,
+            detail,
+            invalidate_cache,
+            delimiter,
+            return_glob,
+            **kwargs,
+        )
+
+    def find(self, path, withdirs=False, prefix="", **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.find(az_path, withdirs, prefix, **kwargs)
+
+    def mkdir(self, path, exist_ok=False):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.mkdir(az_path, exist_ok)
+
+    def rmdir(self, path: str, delimiter="/", **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.rmdir(az_path, delimiter, **kwargs)
+
+    def size(self, path):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.size(az_path)
+
+    def isfile(self, path):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.isfile(az_path)
+
+    def isdir(self, path):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.isdir(az_path)
+
+    def exists(self, path):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.exists(az_path)
+
+    def cat(self, path, recursive=False, on_error="raise", **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.cat(az_path, recursive, on_error, **kwargs)
+
+    def url(self, path, expires=3600, **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.url(az_path, expires, **kwargs)
+
+    def expand_path(self, path, recursive=False, maxdepth=None):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.expand_path(az_path, recursive, maxdepth)
+
+    def rm(self, path, recursive=False, maxdepth=None, **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.rm(az_path, recursive, maxdepth, **kwargs)
+
+    def open(self, path, mode="rb", block_size=None, cache_options=None, **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.open(az_path, mode, block_size, cache_options, **kwargs)
+
+    def touch(self, path, truncate=True, **kwargs):
+        az_path = self._convert_wasb_schema_to_az(path)
+        return self.azureBlobFS.touch(az_path, truncate, **kwargs)
+
+    def _azureBlobFS(self):
+        return self.azureBlobFS
```

Comment:
what you are trying to achieve here? I think this is redundant

Reply:
Done